package.json

```diff
@@ -11,6 +11,11 @@
     ]
   },
   "scripts": {
+    "@TODO.1": "Upgrade or remove these blocks and remove the --ignore-package options (also @TODO.2)",
+    "@TODO.2": "Upgrade or remove these blocks and remove the --ignore-package options (also @TODO.1)",
```

Comment:
Do these `@TODO` scripts make sense out of order?

Reply:
Nope, not really, but I have not investigated why suddenly the package.json is ordered (the order is enforced)

Reply:
They are gone (I removed them in another PR as they were outdated)